src/master_element/Tri32DCVFEM.C

```diff
@@ -599,6 +599,21 @@ void Tri32DSCS::shifted_grad_op(
 //--------------------------------------------------------------------------
 //-------- face_grad_op ----------------------------------------------------
 //--------------------------------------------------------------------------
+void Tri32DSCS::face_grad_op(
+  int /*face_ordinal*/,
+  SharedMemView<DoubleType**>& coords,
+  SharedMemView<DoubleType***>& gradop)
+{
+  using traits = AlgTraitsEdge2DTri32D;
+
+  constexpr int derivSize = traits::numFaceIp_ * traits::nodesPerElement_ * traits::nDim_;
+  DoubleType psi[derivSize];
```

Comment:
Does this need an align?

Reply:
I can combine generic_grad_op_2d with generic_grad_op_3d so it is just generic_grad_op and works for all dimensions.  Then the same code will be used for all elements. 

Seems like when in doubt align is the thing to do.  I'll add it.